v2/env

```diff
@@ -1,6 +1,6 @@
 MAJOR_VERSION=v2
-BASE=24.04
+BASE=22.04
```

Comment:
So, this is set here, but it is not used in the Dockerfile? See my other comment

Reply:
Ok, I have commit suggested change.